src/processor.rs

```diff
@@ -1,3 +1,5 @@
+use std::result;
+
 use textdistance::nstr::damerau_levenshtein;
```

Comment:
Remove unnecessary modules.

```suggestion
use textdistance::nstr::damerau_levenshtein;
```